protoc-gen-orion/orion.go

```diff
@@ -75,6 +78,11 @@ type handler struct {
 	MethodName string
 	Path       string
 }
+type method struct {
+	ApiID       string
+	ServiceName string
```

Comment:
Use `SvcName` to be consistent with other types

Reply:
good catch. updated!